tutorials/Python/neonUtilities/neonUtilitiesPython.md

```diff
@@ -0,0 +1,342 @@
+    11: Brazil (SP 1) [https]            12: Brazil (SP 2) [https]          
+    13: Bulgaria [https]                 14: Chile 1 [https]                
+    15: Chile 2 [https]                  16: China (Guangzhou) [https]      
+    17: China (Lanzhou) [https]          18: China (Shanghai) [https]       
+    19: Colombia (Cali) [https]          20: Czech Republic [https]         
+    21: Denmark [https]                  22: East Asia [https]              
+    23: Ecuador (Cuenca) [https]         24: Ecuador (Quito) [https]        
+    25: Estonia [https]                  26: France (Lyon 1) [https]        
+    27: France (Lyon 2) [https]          28: France (Marseille) [https]     
+    29: France (Montpellier) [https]     30: France (Paris 2) [https]       
+    31: Germany (Erlangen) [https]       32: Germany (Göttingen) [https]    
+    33: Germany (Münster) [https]        34: Greece [https]                 
+    35: Iceland [https]                  36: Indonesia (Jakarta) [https]    
+    37: Ireland [https]                  38: Italy (Padua) [https]          
+    39: Japan (Tokyo) [https]            40: Japan (Yonezawa) [https]       
+    41: Malaysia [https]                 42: Mexico (Mexico City) [https]   
+    43: Norway [https]                   44: Philippines [https]            
+    45: Serbia [https]                   46: Spain (A Coruña) [https]       
+    47: Spain (Madrid) [https]           48: Sweden [https]                 
+    49: Switzerland [https]              50: Turkey (Denizli) [https]       
+    51: Turkey (Mersin) [https]          52: UK (Bristol) [https]           
+    53: UK (Cambridge) [https]           54: UK (London 1) [https]          
+    55: USA (CA 1) [https]               56: USA (IA) [https]               
+    57: USA (KS) [https]                 58: USA (MI 1) [https]             
+    59: USA (NY) [https]                 60: USA (OR) [https]               
+    61: USA (TN) [https]                 62: USA (TX 1) [https]             
+    63: Vietnam [https]                  64: (other mirrors)                
+    
+    
+    
+    
+    
+    
+    Selection: 56
+    
+    
+    
+    The downloaded binary packages are in
+    	/var/folders/_k/gbjn452j1h3fk7880d5ppkx1_9xf6m/T//Rtmpp3bvLx/downloaded_packages
+    
+    
+    
+
+
+Now import the `devtools` package. This does need to be run every time you use the code; for those of you who are familiar with R, `importr` is roughly equivalent to the `library()` function in R.
+
+
+```python
+devtools = importr('devtools')
+```
+
+Using the `devtools` package, install the `neonUtilities` package. As with `devtools`, you can theoretically skip this step in the future, but since we update `neonUtilities` frequently, we recommend re-installing regularly.
+Then import the package.
+
+
+```python
+devtools.install_github('NEONScience/NEON-utilities/neonUtilities');
+neonUtils = importr('neonUtilities')
+```
+
+## Stack data files
```

Comment:
For the neonUtilities lesson the headers were the function names (`stackBytable()`). It would be nice to be consistent but I could go either way: name (as in neonUtilities) or purpose (as here). Thoughts?

Reply:
Good point. I like both. Possibly we could change both tutorials to something like "Stack data files with stackByTable()". Or is that too wordy?

Reply:
We could try.  "Combine Data Files: stackByTable()"?  Ideally we want to keep it to one line even on smaller desktop screens so the sidebar menu doesn't look too weird. See: http://www.neonscience.org/neon-api-usage